CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+project(libMCNP2CAD)
+
+# cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 2.8)
+
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -DHAVE_IGEOM_CONE" )
+
+# adjust compiler setting for Linux using gcc version 5.0 and higher
+if(CMAKE_SYSTEM_NAME MATCHES Linux AND CMAKE_COMPILER_IS_GNUCXX)
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "5.0")
+    if(NOT CMAKE_CXX_FLAGS MATCHES _GLIBCXX_USE_CXX11_ABI)
+      set(CMAKE_CXX_FLAGS "-D_GLIBCXX_USE_CXX11_ABI=0 ${CMAKE_CXX_FLAGS}")
+    endif()
+  endif()
+endif()
+
+# Get the packages needed to use the SDK
+find_package(Cubit REQUIRED CONFIG)
```

Comment:
but this will now be a stand-alone library that needs to know where to find iGeom and hopefully upon doing so, is told where to find the other includes